src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -874,6 +874,18 @@ void ConnectionHandlerImpl::CreateWebEngineDevice() {
   transport_manager_.CreateWebEngineDevice();
 }
 
+bool ConnectionHandlerImpl::IsAppPresentInConnections(
+    const uint32_t connection_key) const {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  transport_manager::ConnectionUID connection_handle = 0;
+  uint8_t session_id = 0;
+  PairFromKey(connection_key, &connection_handle, &session_id);
+  auto connection_uid = ConnectionUIDFromHandle(connection_handle);
+
+  return transport_manager_.IsApplicationPresentInConnections(connection_uid);
```

Comment:
@ydementieiev Is it possible to use `ConnectionHandler`'s connections list instead of `TransportManager`'s one?

Reply:
@mvorobio I reworked, [please check](https://github.com/LuxoftSDL/sdl_core/pull/44/commits/48461efa6f508175571a07e0570289ad4d6f5ed3)